tutorials/introductory/usage.py

```diff
@@ -175,77 +170,373 @@
 plt.legend()
 
 ###############################################################################
-# In addition, there is a third approach, for the case when embedding
+# (In addition, there is a third approach, for the case when embedding
 # Matplotlib in a GUI application, which completely drops pyplot, even for
-# figure creation.  We won't discuss it here; see the corresponding section in
-# the gallery for more info (:ref:`user_interfaces`).
+# figure creation. See the corresponding section in the gallery for more info
+# (:ref:`user_interfaces`).)
 #
 # Matplotlib's documentation and examples use both the OO and the pyplot
-# approaches (which are equally powerful), and you should feel free to use
-# either (however, it is preferable to pick one of them and stick to it, instead
-# of mixing them).  In general, we suggest to restrict pyplot to interactive
-# plotting (e.g., in a Jupyter notebook), and to prefer the OO-style for
-# non-interactive plotting (in functions and scripts that are intended to be
-# reused as part of a larger project).
+# styles.  In general, we suggest using the OO style, particularly for
+# complicated plots, and functions and scripts that are intended to be reused
+# as part of a larger project. However, the pyplot style can be very conveneient
+# for quick interactive work.
 #
 # .. note::
 #
-#    In older examples, you may find examples that instead used the so-called
-#    ``pylab`` interface, via ``from pylab import *``. This star-import
-#    imports everything both from pyplot and from :mod:`numpy`, so that one
-#    could do ::
-#
-#       x = linspace(0, 2, 100)
-#       plot(x, x, label='linear')
-#       ...
+#    You may find older examples that use the ``pylab`` interface,
+#    via ``from pylab import *``. This approach is strongly deprecated.
 #
-#    for an even more MATLAB-like style.  This approach is strongly discouraged
-#    nowadays and deprecated. It is only mentioned here because you may still
-#    encounter it in the wild.
+# Making a helper functions
+# -------------------------
 #
-# If you need to make the same plots over and over
-# again with different data sets, use the recommended signature function below.
+# If you need to make the same plots over and over again with different data sets,
+# or want to easily wrap Matplotlib methods, use the recommended signature function
+# below.
 
 
 def my_plotter(ax, data1, data2, param_dict):
     """
-    A helper function to make a graph
+    A helper function to make a graph.
```

Comment:
I have mixed feeling about dropping the docstring.  On one hand, I see the reason to make it terser, on the other hand we should model best practices.

Reply:
I don't feel strongly about this, and am happy to add it back in.  However, I think for a little in-script utility its OK to not have a numpydoc compliant docstring, and I found it distracting for a tutorial.